web/src/app/services/user.service.ts

```diff
@@ -57,11 +57,30 @@ export class UserService {
   }
 
   addRefID(user): UserSocial {
-    const normalisedResponse = { ...user.additionalUserInfo.profile, reference: user.uid };
+    const normalisedResponse = { ...user.additionalUserInfo.profile, userId: user.uid };
     return normalisedResponse;
   }
 
-  saveUserSocial(social): Observable<UserSocial> {
+  checkForSocialDoc(provider: string, userId: string): Observable<any> {
```

Comment:
In the observable do we know the `any`? I prefer not to use `any` if possible

Reply:
@eddiejaoude As the returned object is constructed dynamically inside the function I used any as datatype there. Shall we create a static model for the object ?

Reply:
Is this function not the same as? https://github.com/DashboardHub/TribeDashboard/blob/00b4bec91cd43a273169dd405383c1d4af17d678/web/src/app/services/user.service.ts#L42

Reply:
Also the variable names are not consistent, here it is `provider` but else where in the file it is `providerId`, same with `userId` and `uid` - I dont mind too much, as long as we are consistent 🤓

Reply:
Okay @eddiejaoude I will make it consistent. The query is same but the payload returned is different, that's why I wrote separate functions.

Reply:
This returned data is different but is available in the other function d returned data? Looking at where you are using it, I don't think a whole new method is required, unless I am missing something? https://github.com/DashboardHub/TribeDashboard/pull/59/files/00b4bec91cd43a273169dd405383c1d4af17d678#diff-0f5867bb66d30741330d96dea4362655R35